topsail/testing/regression.py

```diff
@@ -0,0 +1,168 @@
+import os
+import json
+
+import numpy as np
+from functools import reduce
+
+from opensearchpy import OpenSearch
+
+regression_return_codes = {
+    "POS_REG": 1,
+    "NEG_REG": -1,
+    "NONE": 0,
+    "UNDEF": 2,
+}
```

Comment:
can you turn this into a Python enum, and make sure that the string ends up in the final JSON (not the int)
and if you can use full words instead of abbreviations :)

Reply:
@kpouget Thanks for the feedback! I actually thought I closed this PR, this was a precursor to [the matbench PR](https://github.com/openshift-psap/matrix-benchmarking/pull/125). This is all old code and has been adapted into matrix-benchmarking :)